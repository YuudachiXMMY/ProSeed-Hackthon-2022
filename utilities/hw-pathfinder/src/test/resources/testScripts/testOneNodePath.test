# Test Dijkstra's Algorithm in a one node path

CreateGraph path

AddNode path A

FindPath path A A
