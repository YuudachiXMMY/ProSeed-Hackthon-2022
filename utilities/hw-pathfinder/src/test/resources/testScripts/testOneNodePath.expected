# Test Dijkstra's Algorithm in a one node path

created graph path

added node A to path

path from A to A:
total cost: 0.000